components/modal/style/modal.less

```diff
@@ -133,6 +134,42 @@
   }
 }
 
+.vertical-center-modal {
+  text-align: center;
+  &:before {
+    content: '';
+    display: inline-block;
+    height: 100%;
+    vertical-align: middle;
+    width: 0;
+  }
+  .@{dialog-prefix-cls} {
+    display: inline-block;
+    vertical-align: middle;
+    top: 0;
+    text-align: left;
+  }
+  .@{table-prefix-cls} {
+    &-body {
+      white-space: nowrap;
+      overflow-x: auto;
+    }
```

Comment:
Why adding this?

Reply:
The table in modal, `white-space:nowrap;` horizontal scrolling seems to experience better, or do you mean that it is better controlled by the table itself?